.gitignore

```diff
@@ -101,3 +101,5 @@ node_modules
 # Ignore public folder (used for local document uploads)
 public
 
+# Ignore Vets API Mockdata
+vets-api-mockdata/
```

Comment:
Are you editing mockdata locally on the fly? Trying to see how this is just being added now and why.